bootstrap.sh

```diff
@@ -14,7 +14,7 @@ kubectl apply -f .infrastructure/app/clusterIp.yml
 kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
 kubectl apply -f .infrastructure/app/deployment.yml
-
+kubectl apply -f .infrastructure/security/rbac.yml
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
 # kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
The line for applying the ingress configuration is commented out. If the ingress is needed for your application, ensure to uncomment this line. Otherwise, if it's intentional, you can leave it as is.